sklearn/decomposition/tests/test_dict_learning.py

```diff
@@ -57,6 +57,54 @@ def test_dict_learning_overcomplete():
     assert dico.components_.shape == (n_components, n_features)
 
 
+def test_max_iter():
+    def ricker_function(resolution, center, width):
+        """Discrete sub-sampled Ricker (Mexican hat) wavelet"""
+        x = np.linspace(0, resolution - 1, resolution)
+        x = ((2 / ((np.sqrt(3 * width) * np.pi ** 1 / 4)))
+             * (1 - ((x - center) ** 2 / width ** 2))
+             * np.exp((-(x - center) ** 2) / (2 * width ** 2)))
```

Comment:
You have extraneous brackets here making things hard to read. ** takes precedence over *